RealEstate/apps/core/static/style.css

```diff
@@ -0,0 +1,122 @@
+/* common */
+
+ul.list-group {
+	padding-top:30px;
+}
+#heading {
+	padding-bottom: 30px;
+}
+.icon-ok {
+	display: none;
+	font-size: 80%;
+	color: green;
+	padding-left: 20px
+}
+.icon-delete {
+    display: none;
+    font-size: 90%;
+    color: #A94431;
+    padding-left: 20px;
+}
+.icon-edit {
+    display: none;
+    color: #31708f;
+    padding-left: 20px;
+    cursor: pointer;
+    font-size: 90%;
+}
+.vcenter {
+	display: inline-block;
+	vertical-align: middle;
+	float: none;
+	margin-left:-3px;
+}
+.inner-rating {
+	padding:20px 0;
+	padding-left:10%
+}
+.slider-selection {
+	background: #BABABA;
+}
+#helptext-L{
+	padding-right: 15px;
+}
+#helptext-R{
+	padding-left: 15px;
+}
+#button-margin{
+	margin:30px;
+}
```

Comment:
We have a lot of these margins on the buttons. They're all essentially the same. What if we got rid of the id's and classes and just said:
```button {
   margin: 30px;
}```

Might make everything more consistent and clean?